applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_interface_integration_coefficients.cpp

```diff
@@ -0,0 +1,57 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//                   Gennady Markelov
+//
+
+#include "custom_elements/integration_coefficients_calculator.h"
+#include "custom_geometries/line_interface_geometry.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+#include "tests/cpp_tests/test_utilities/model_setup_utilities.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+using namespace std::string_literals;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(InterfaceIntegrationCoefficients_ReturnsCorrectValue, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Set
+    const auto interface_integration_coefficients = IntegrationCoefficientsCalculator{};
+    // The shape function values for this integration point are 0.2, 0.5 and 0.3 for nodes 1, 2 and 3 respectively
+    const Geometry<Node>::IntegrationPointType       integration_point(0.5, 0.3, 0.0, 0.5);
```

Comment:
If I'm correct the actual location doesn't really matter, so this should be equivalent:
```suggestion
    const Geometry<Node>::IntegrationPointType       integration_point(0.0, 0.0, 0.0, 0.5);
```

Reply:
indeed, changed here and other places